config/overlays/odh/params.env

```diff
@@ -2,3 +2,5 @@ odh-mm-rest-proxy=quay.io/opendatahub/rest-proxy:fast
 odh-modelmesh-runtime-adapter=quay.io/opendatahub/modelmesh-runtime-adapter:fast
 odh-modelmesh=quay.io/opendatahub/modelmesh:fast
 odh-modelmesh-controller=quay.io/opendatahub/modelmesh-controller:fast
+ose-etcd=registry.redhat.io/openshift4/ose-etcd-rhel9:v4.19
+ose-cli=registry.redhat.io/openshift4/ose-cli-rhel9:v4.19
```

Comment:
Any particular reason for adding it?

Reply:
@spolti the idea here is that of [RHOAIENG-25619](https://issues.redhat.com/browse/RHOAIENG-25691), where the images can be kept consistent with the rest of RHOAI, rather than the old approach of every component specifying their own version of everything, and those all going stale and having 7 different versions of the same image across the different components.

Reply:
@grdryn I am confused now, will ose-cli replace ose ose-proxy? If not, why add ose-cli?

Reply:
But, if the question I missed is about standardizing the ose-proxy, then yes, we can do the same as done in the PR for etcd.

Reply:
@spolti ose-cli is a different case than oauth-proxy. The idea is to keep _all_ 3rd party images (or at least other images from RH registry) up-to-date, so ose-cli is being added because it's referenced in config/overlays/odh/quickstart.yaml on L52 (and you can see the reference is updated in this PR).

So maybe we should have a 3rd entry here for ose-oauth-proxy? Or I guess that would actually need to be passed to that deployment.yaml.tmpl from [here](https://github.com/opendatahub-io/modelmesh-serving/blob/7e8b60b7d3486347242cfb15383f961240fb0b3b/controllers/modelmesh/modelmesh.go#L98) 🤔 Maybe we should update the static reference once in this PR, and then consider parameterizing that and figuring out how to feed the image through in a potential follow-up change?

Reply:
I see, thanks.

Yes, this template is processed at runtime, we could add the image there and later patch it accordingly.

Reply:
Shall we already add the proxy as part of this PR?

Reply:
I'm easy with it either being done in this PR or another one. Which would you prefer? Maybe we could get this one merged, then create a follow up one for that (or potentially 2 follow ups: one to just update the static image reference in place, then a second to have the image passed through, if that's even a reasonable thing to do)?